easybuild/easyconfigs/f/fsm-lite/fsm-lite-1.0-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,34 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = "MakeCp"
+
+name = 'fsm-lite'
+version = '1.0'
+
+homepage = "https://github.com/nvalimak/fsm-lite"
+description = """A singe-core implemetation of frequency-based substring mining."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = ['https://github.com/nvalimak/%(name)s/archive/']
+sources = ['v%(version)s-stable.tar.gz']
+checksums = ['f781a9fbab5265bd09b3b5b7e1cba904582ec201c3d30baed36e28a03de3ac61']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('binutils', '2.39'),
+]
+
+dependencies = [
+    ('sdsl-lite', '2.0.3'),
+]
+
+prebuildopts = "sed -i '1s/.*/SDSL_INSTALL_PREFIX=${EBROOTSDSLMINLITE}/' Makefile && make depend &&"
+
+files_to_copy = [(['fsm-lite'], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/fsm-lite'],
+    'dirs': [],
+}
```

Comment:
@verdurin is it possible to add a `sanity_check_commands` item here?

Reply:
@smoors done